src/System.Management.Automation/minishell/api/RunspaceConfigurationTypeException.cs

```diff
@@ -20,12 +20,7 @@ namespace System.Management.Automation.Runspaces
     ///     2. ISerializable
     /// -->
     [Serializable]
-#if CORECLR
-    internal
-#else
-    public
-#endif
-    class RunspaceConfigurationTypeException : SystemException, IContainsErrorRecord
+    internal class RunspaceConfigurationTypeException : SystemException, IContainsErrorRecord
```

Comment:
Say you already have a branch `'cleanup-runspaceconfigurationtypeexception'` making method signature cleanup. Then when working on `RunspaceConfigurationTypeAttribute.cs`, you find the change there is the same kind as in former branch, then make the change in the existing branch, so that when submitting PR from the former branch, the changes contain both `runspaceconfigurationtypeexception.cs` and `RunspaceConfigurationTypeAttribute.cs`.

This basically means 2 things:
1. you don't need to be exhaustive (no need to have a single PR contains the same kind of changes for the whole code base). Just make sure for the files you go through in one iteration, changes of the same kind are grouped in one PR.
2. don't submit a PR right after making changes to a single file. Instead, finish the iteration (say you go through 10 files for tonight), then submit PRs. This is easier for you to track what changes should be grouped together.